community/io/src/main/java/org/neo4j/io/pagecache/impl/CompositePageCursor.java

```diff
@@ -448,6 +448,12 @@ public int copyTo( int sourceOffset, PageCursor targetCursor, int targetOffset,
         throw new UnsupportedOperationException( "Composite cursor does not support copyTo functionality." );
     }
 
+    @Override
+    public void shiftBytes( int sourceOffset, int length, int shift )
+    {
+        throw new UnsupportedOperationException( "Composite cursor does not support copyTo functionality." );
```

Comment:
Why not? Or it just doesn't support it _yet_ since there's no need?

Reply:
Exactly! Updating copied error message.